pytest_examples/find_examples.py

```diff
@@ -140,7 +140,7 @@ def find_examples(*paths: str | Path, skip: bool = False) -> Iterable[CodeExampl
             group = uuid4()
             if path.suffix == '.py':
                 code = path.read_text('utf-8')
-                for m_docstring in re.finditer(r'(^ *)(r?"""\n)(.+?)\1"""', code, flags=re.M | re.S):
+                for m_docstring in re.finditer(r'(^ *)(r?"""[^\r\n]*\n)(.+?)\1"""', code, flags=re.M | re.S):
```

Comment:
```suggestion
                for m_docstring in re.finditer(r'(^ *)(r?""".*?\n)(.+?)\1"""', code, flags=re.M | re.S):
```

Should work the same.

Reply:
Thanks! Updated in [85681d7](https://github.com/pydantic/pytest-examples/pull/7/commits/85681d7cb97b477ceddf5daf17811a37e415cff7)